docs/get-involved/delegation-policy.md

```diff
@@ -1,14 +1,42 @@
 # Delegation Policy
 
-Oasis Protocol Foundation delegates ROSE tokens to node operators based on their
-participation in the Oasis Mainnet, Testnet and ParaTimes on the network,
+Oasis Protocol Foundation (OPF) delegates ROSE tokens to node operators based on
+their participation in the Oasis Mainnet, Testnet and ParaTimes on the network,
 node reliability and performance, community engagement and overall support of
 the network.
 
-This page describes the Oasis Protocol Foundation's Delegation Policy.
+:::danger
+
+Starting in the first quarter of 2024, Oasis Protocol Foundation (OPF) will
+initiate the Know Your Customer (KYC) process for all validators receiving
+delegation from OPF. It's essential for Node Operators to ensure that you have
+access to the email address provided in yours metadata submissions. This step
+is crucial for completing the KYC procedure efficiently. Please update your
+contact details if necessary.
+
+:::
 
 ## Requirements for Receiving Delegations
 
+### How to Earn Delegations?
+
+1. Join the `#node-operators` channel on [Oasis Discord] server
+2. Create your Testnet entity key and Mainnet [entity key]
+3. [Configure and start] Testnet consensus and paratime nodes
+4. [Configure and start] Mainnet consensus node and paratime nodes that you are
+able to  start
+5. [Submit metadata] for Mainnet and Testnet
+6. Apply for a delegation and pass KYC
```

Comment:
Where to apply?

Reply:
I added a link to the form, but it would be nice if someone reviewed the form.